shed/event_streams.py

```diff
@@ -735,7 +737,7 @@ def update(self, x, who=None):
             tup = tuple(buf.popleft() for buf in self.buffers)
             self.condition.notify_all()
             self.prior = tup
-            return self.emit(tup)
+            return self.emit(self.curate_streams(tup))
```

Comment:
I think we'll need to think more carefully on that one. Simply decorating may not work, since some nodes assume that nds is `name, doc` whereas others expect `name, docs` where `docs` is a tuple (even if length 1).
Maybe it is good to make it another PR. Normalizing all the inputs/outputs of the streams altogether.
It could supersede this one (so we don't merge this)